lib/toAscii.js

```diff
@@ -27,7 +27,12 @@ const postReplacements = [
   [ /MØI/ug, 'µØI' ], // Use centered tippi on top of spacer char
   [ /([@R®H´ÍÏçœ˜†])u/ug, '$1ü' ], // Use lower aunkar when char is pair akhar
   [ /([@R®H´ÍÏçœ˜†])U/ug, '$1¨' ], // Use lower dulankaar when char is pair akhar
-  [ /(.)iÎ/ug, 'i$1Î' ], // Swap i with Î
+  [ 'sæ', 'S' ],
```

Comment:
Why are these post replacements and not mappings?

Reply:
They should be in `unicodeMappings.json`, however, it doesn't allow for multiple ASCII mappings.

For example, `S` would need to be mapped twice, but that is not allowed.

Reply:
Ok. To make it extra clear that they're not post-replacement rules, could you extract them into a separate mappings array and then spread it into the final list?

Reply:
Sure